lessons/intro/cython/index.md

```diff
@@ -491,7 +490,7 @@ Ke každému řádku se navíc dá kliknutím ukázat odpovídající kód v C (
 protože řeší věci jako zpětnou kompatibilitu a ošetřování chyb, a navíc používá hodně pomocných
 maker).
 
-Obecně nebývá problém mít "žluté" řádky na úrovni funkce, kde se provádí pouze jednou.
+Obecně nebývá problém mít "žluté" řádky na úrovni funkce, které se provádí pouze jednou.
```

Comment:
> funkce, která...

ale celé to tady zní dost kostrbatě, těžko říct, co jsme tím chtěli říct. Co třeba:

> Obecně nebývá problém mít „žluté“ ty řádky, které se ve funkci provádí pouze jednou.